packages/toast/toast.scss

```diff
@@ -0,0 +1,216 @@
+
+        .jkl-countdown {
+            border-radius: 0;
+        }
+    }
+
+    &__icon {
+        grid-area: icon;
+        width: jkl.$spacing-24;
+        margin-right: jkl.$spacing-16;
+        @include jkl.forced-colors-svg-fallback($stroke: CanvasText);
+    }
+
+    &__content {
+        grid-area: content;
+    }
+
+    &__message {
+        margin-top: jkl.rem(-2px); // Offset for å aligne med ikon hvis uten tittel.
+    }
+
+    &__title {
+        @include jkl.text-style("heading-4");
+        margin: 0 0 jkl.$spacing-8 0;
+
+        & ~ .jkl-toast__message {
+            margin-top: 0; // Unset offset satt for å aligne med ikon hvis uten tittel.
+        }
+    }
+
+    &__dismiss-button {
+        @include jkl.reset-outline;
+        grid-area: dismiss;
+        justify-self: end;
+        position: relative;
+        margin-left: var(--jkl-toast-gap);
+
+        background-color: transparent;
+        border-radius: jkl.rem(3px);
+        padding: 0;
+        cursor: pointer;
+
+        color: inherit;
+        @include jkl.forced-colors-svg-fallback($stroke: ButtonText);
+        @include jkl.forced-colors-mode {
+            background-color: ButtonFace;
+        }
+
+        // Sørg for å ha en stor nok touch target.
+        &::after {
+            --tap-increment: calc((#{jkl.rem(44px)} - #{jkl.rem(20px)}) / 2 * -1);
+            content: "";
+            position: absolute;
+            inset: var(--tap-increment) var(--tap-increment) var(--tap-increment) var(--tap-increment);
+        }
+
+        &:focus {
+            @include jkl.keyboard-navigation {
+                outline: 2px solid var(--jkl-color);
+
+                @include jkl.forced-colors-mode {
+                    outline: 2px solid ButtonText;
+                    outline-offset: 2px;
+                }
+            }
+        }
+    }
+
+    &--info {
+        color: jkl.$color-granitt;
+        background-color: colors.varslingsfarge("info");
+    }
+
+    &--warning {
+        color: jkl.$color-granitt;
+        background-color: colors.varslingsfarge("advarsel");
+    }
+
+    &--error {
+        color: jkl.$color-granitt;
+        background-color: colors.varslingsfarge("feil");
+    }
+
+    &--success {
+        color: jkl.$color-granitt;
+        background-color: colors.varslingsfarge("suksess");
+    }
+
+    @include jkl.forced-colors-mode {
+        border: 2px solid CanvasText;
+
+        &--info {
+            border-style: dotted;
+        }
+
+        &--warning {
+            border-style: dashed;
+        }
+
+        &--error {
+            border-style: double;
+            border-width: 4px;
+        }
+    }
+}
+
+$_entering: jkl-entering-#{string.unique-id()};
+$_exiting: jkl-exiting-#{string.unique-id()};
+
+.jkl-toast[data-animation="entering"],
+.jkl-toast[data-animation="queued"] {
+    animation: $_entering jkl.timing("productive") jkl.easing("entrance") forwards;
```

Comment:
Tror kanskje jeg ville ha foretrukket en litt lenger animasjon (f.eks. `expressive`), i hvert fall på entry; det tiltrekker seg litt mer oppmerksomhet og føles hakket _smoothere_. Toast vil jo ofte dukke opp uten at brukeren venter det/har initiert det, så det er ikke nødvendigvis et krav at ting skjer superfort :)

Reply:
`expressive` ble for treg, men Lou og jeg er enige i at `productive` er for rask. Vi legger til et steg i skalaen på 200ms.